otelcollector/shared/configmap/mp/tomlparser-default-scrape-settings.go

```diff
@@ -199,6 +220,9 @@ func (fcw *FileConfigWriter) WriteDefaultScrapeSettingsToFile(filename string, c
 	file.WriteString(fmt.Sprintf("AZMON_PROMETHEUS_ACSTORCAPACITYPROVISIONER_SCRAPING_ENABLED=%v\n", cp.AcstorCapacityProvisioner))
 	file.WriteString(fmt.Sprintf("AZMON_PROMETHEUS_ACSTORMETRICSEXPORTER_SCRAPING_ENABLED=%v\n", cp.AcstorMetricsExporter))
 	file.WriteString(fmt.Sprintf("AZMON_PROMETHEUS_LOCALCSIDRIVER_SCRAPING_ENABLED=%v\n", cp.LocalCSIDriver))
+	file.WriteString(fmt.Sprintf("AZMON_PROMETHEUS_ZTUNNEL_SCRAPING_ENABLED=%v\n", cp.Ztunnel))
```

Comment:
these vars need to be added here so that we can determine if it is enabled on a cluster using telemetry - https://github.com/Azure/prometheus-collector/blob/690de4bae1a8c2fa5323cd8af38e93cc90c705d1/otelcollector/fluent-bit/src/process_stats.go#L35